Packet++/src/BgpLayer.cpp

```diff
@@ -76,12 +76,15 @@ std::string BgpLayer::getMessageTypeAsString() const
 	}
 }
 
-void BgpLayer::parseNextLayer()
+void BgpLayer::parseNextLayer(ProtocolType parseUntil, OsiModelLayer parseUntilLayer)
 {
 	size_t headerLen = getHeaderLen();
 	if (m_DataLen <= headerLen || headerLen == 0)
 		return;
 
+	if (getProtocol() == parseUntil || getOsiModelLayer() == parseUntilLayer)
+        return;
```

Comment:
It will be a pain to change a bit but all `return` statements in cpp files unfortunately requires a tab indentation

Reply:
Got it, somehow didn’t notice that locally, I’ll go through and add them